assets/scss/base/_lists.scss

```diff
@@ -1,38 +1,96 @@
+/*
+Lists
 
-/**
-* @desc         lists.
-* @name         layouts
-* @author       Anthony Munene
-* @tested       N/A
-* @requires     N/A
+
+Markup:
+
+<ul>
+  <li class="list-item">
+      <div class="list-item__primary">
+          <h3 class="list-item__heading">Sample heading</h3>
+          <p class="list-item__info">Sample info text</p>
+      </div>
+      <div class="list-item__secondary">
+          <a class="list-item__link list-item__link--right" href="review">CTA link</a>
+      </div>
+  </li>
+</ul>
+
+.list-item                - a list item with a bottom grey border
+.list-item__primary       - primary content of list item
+.list-item__secondary     - secondary content of list item
+.list-item__heading       - heading of list item
+.list-item__info          - list item information
+.list-item__info--right   - list item information right aligned
+.list-item__link          - link within list item
+.list-item__link--right   - link within list item rlight aligned
+
+Styleguide Lists
 */
 
+.list-item {
+  display: table;
+  width: 100%;
+  border-bottom: 1px solid $border-colour;
+}
+
+.list-item__primary,
+.list-item__secondary {
+  display: table-cell;
+  vertical-align: top;
+}
+
+.list-item__heading {
+  @include core-19;
+  display: inline-block;
+  margin: 0;
+}
+
+.list-item__info {
+  @extend .faded-text;
+  @include core-16;
+  margin-top: em(5) !important;  // neeed to override .content__body p
+}
+
+.list-item__link {
+  @include core-19;
+}
+
+.list-item__info--right,
+.list-item__link--right {
```

Comment:
If I understand right can you use `{text-align: right}` on `.list-item__secondary` instead of `.list-item__link--right` and `.list-item__info--right` ?